examples/content-source-maps-cpa/src/pages/index.tsx

```diff
@@ -1,21 +1,21 @@
 import { GetStaticProps } from 'next';
-import { Post, getAllPostsForHome } from '../../lib/api-rest';
+import { Post as PostType, getAllPostsForHome } from '../../lib/api-rest';
+import { useContentfulLiveUpdates } from '@contentful/live-preview/react';
+
+const Post = ({ post }: { post: PostType }) => {
+  const updatedPost = useContentfulLiveUpdates(post);
```

Comment:
you will only get updates for post, but not if a new one is added should we move it one level up?